<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>DNAStatMSAProfileDialog</class>
 <widget class="QDialog" name="DNAStatMSAProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Alignment Profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Profile mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="countsRB">
        <property name="text">
         <string>Counts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="percentsRB">
        <property name="text">
         <string>Percents</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Custom options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="gapCB">
        <property name="text">
         <string>Show scores for gaps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unusedCB">
        <property name="text">
         <string>Show scores for symbols not used in alignment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="skipGapPositionsCB">
        <property name="text">
         <string>Skip gaps in consensus position increments</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="saveBox">
     <property name="title">
      <string>Save profile to file</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fileEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="fileButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="htmlRB">
        <property name="text">
         <string>Hypertext  (.HTML)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="csvRB">
        <property name="text">
         <string>Comma separated ( .CSV)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DNAStatMSAProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DNAStatMSAProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
